core/src/main/scala/org/apache/spark/rpc/netty/NettyRpcEnv.scala

```diff
@@ -47,11 +48,15 @@ private[netty] class NettyRpcEnv(
     host: String,
     securityManager: SecurityManager,
     numUsableCores: Int) extends RpcEnv(conf) with Logging {
+  val role = conf.get(EXECUTOR_ID).flatMap { id =>
+    if (id == SparkContext.DRIVER_IDENTIFIER) Some("driver") else Some("executor")
+  }
 
   private[netty] val transportConf = SparkTransportConf.fromSparkConf(
     conf.clone.set(RPC_IO_NUM_CONNECTIONS_PER_PEER, 1),
     "rpc",
-    conf.get(RPC_IO_THREADS).getOrElse(numUsableCores))
+    conf.get(RPC_IO_THREADS).getOrElse(numUsableCores),
+    role)
```

Comment:
This seems not necessary to separate into another line.

Reply:
This is fine (one arg per line).

Reply:
@jerryshao , The guide says,

> For method declarations, use 4 space indentation for their parameters and put each in each line when the parameters don't fit in two lines.  

I'll keep this code.